.github/workflows/forest.yml

```diff
@@ -363,26 +363,45 @@ jobs:
             ubuntu:focal \
             bash -c "cp /proofs/* /cache"
           sudo chmod -R 755 $FIL_PROOFS_PARAMETER_CACHE
-  offline-rpc-check:
+  calibnet-rpc-checks:
     needs:
       - build-ubuntu
-    name: Offline RPC check
+    name: Calibnet RPC checks
     runs-on: ubuntu-latest
+    env:
+      # We use a custom Dockerfile for CI to speed up the build process.
+      FOREST_DOCKERFILE_OVERRIDE: scripts/devnet/forest_ci.dockerfile
     steps:
-      - run: lscpu
-      - name: Checkout Sources
-        uses: actions/checkout@v4
-      - uses: actions/download-artifact@v4
+      - uses: actions/cache@v4
+        id: cache
         with:
-          name: 'forest-${{ runner.os }}'
-          path: ~/.cargo/bin
+          path: '${{ env.FIL_PROOFS_PARAMETER_CACHE }}'
+          key: proof-params-keys
+      - name: Load cache into volume
+        if: steps.cache.outputs.cache-hit == 'true'
+        run: |
+          docker volume create api_compare_filecoin-proofs
+          docker run --rm \
+            -v api_compare_filecoin-proofs:/proofs \
+            -v $FIL_PROOFS_PARAMETER_CACHE:/cache \
+            ubuntu:focal \
+            bash -c "cp /cache/* /proofs"
```

Comment:
It might make sense to put those in files. It would somehow unclutter the yaml files (less important), but it will allow `shellcheck` linting, which will enforce good practices. They can even live in `.github/`. What do you think?

Reply:
How about https://github.com/mschuett/yaml-shellcheck

Reply:
https://github.com/ChainSafe/forest/issues/4107